docs/application/native/guides/notification/overview.md

```diff
@@ -9,6 +9,10 @@ You can use the following notifications and content sharing features in your nat
 
   You can keep the user informed of important information with notifications. You can post notifications in the quick panel. You can also control the behavior of the notification by updating its content or showing a progress bar.
 
+- [Notification-ex](notification-ex.md)
```

Comment:
As a new page is added, please update docs/application/toc_all.md file also.
In this file, we list up the location of added pages.
I think this page should be located 
after \#### \[Notifications](/application/native/guides/notification/notifications.md)